monorepo_build.sh

```diff
@@ -28,16 +35,18 @@ for PARAM in $@; do
     if [ "$SUBDIRECTORY" == "" ]; then
         SUBDIRECTORY=$REMOTE
     fi
-    # Rewrite all branches from the first remote, only master branches from others
+
+    echo "Building all branches of the remote '$REMOTE'"
     if [ "$PARAM" == "$1" ]; then
-        echo "Building all branches of the remote '$REMOTE'"
-        $MONOREPO_SCRIPT_DIR/load_branches_from_remote.sh $REMOTE
+        $MONOREPO_SCRIPT_DIR/load_branches_from_remote.sh $REMOTE $SUBDIRECTORY --master
         $MONOREPO_SCRIPT_DIR/rewrite_history_into.sh $SUBDIRECTORY --branches
         MERGE_REFS='master'
     else
-        echo "Building branch 'master' of the remote '$REMOTE'"
+        $MONOREPO_SCRIPT_DIR/load_branches_from_remote.sh $REMOTE $SUBDIRECTORY
```

Comment:
🍖 load & checkout branches from remote for the rest of the remotes; don't check out master a second time